exporters/otlp/README.md

```diff
@@ -0,0 +1,25 @@
+# OTLP Exporter
+
+The [OpenTelemetry Protocol](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/protocol/README.md) (OTLP) is a vendor-agnostic protocol designed as part of the OpenTelemetry project. The OTLP exporter can be used to export to any backend that supports OTLP.
+
+Currently, the only backend that supports OTLP is the [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector). The Collector can be configured to export to many other backends, such as Zipkin and Jaegar.
```

Comment:
Minor: I think we probably want to rephrase this to "The OpenTelemetry Collector is the reference implementation of OTLP backend" so we don't have to worry about whether it is the "only backend at this moment or in the future".